src/style.css

```diff
@@ -1,3 +1,71 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+}
+
 body {
   margin: 0;
+  padding: 0;
+}
+
+header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  margin-right: 419px;
+}
+
+.nav_list {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  list-style: none;
+}
+
+.link_box {
+  margin-right: 20px;
+}
+
+.nav_link {
+  text-transform: uppercase;
+  color: #000;
+  font-size: 12px;
+  text-decoration: auto;
```

Comment:
The `text-decoration` property value `auto` is not valid. Consider using `none`, `underline`, `overline`, `line-through`, or `inherit` instead.